django/core/management/commands/loaddata.py

```diff
@@ -251,6 +251,27 @@ def _get_targets(self, fixture_name, ser_fmt, cmp_fmt):
         )
         return {'.'.join((fixture_name, suffix)) for suffix in suffixes}
 
+    def _find_fixture_files_in_dir(self, fixture_dir, fixture_name, targets):
```

Comment:
We don't need to add the `_` prefix to the new methods. They are considered a private API even without it.

Reply:
Done in c0a62ed4181ae12715eaa341f7a772d829bb9a17